Tests/Marketplace/approved_categories.json

```diff
@@ -8,6 +8,7 @@
 	"Data Enrichment & Threat Intelligence",
 	"Database",
 	"Deception",
```

Comment:
Need to remove the old one

Reply:
should I remove it now or should I open pr with all the old categories and remove them?

Reply:
Yeah its ok, open a PR afterwards to remove all the old categories